src/main/kotlin/com/oneplanet/dto/response/ConstituentGasesResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.oneplanet.dto.response
+
+import io.micronaut.core.annotation.Introspected
+import java.math.BigDecimal
+
+@Introspected
+data class ConstituentGasesResponse(
+    val co2e_total: BigDecimal?, val co2e_other: BigDecimal?, val co2: BigDecimal,
+    val ch4: BigDecimal, val n2o: BigDecimal
```

Comment:
One field per line would be better (see the comment in `EmissionFactorResponse`)